src/standalone/Vegetation/solar_induced_fluorescence.jl

```diff
@@ -110,14 +110,14 @@ function compute_SIF_at_a_point(
     (; kf, kd_p1, kd_p2, min_kd, kn_p1, kn_p2, kp, kappa_p1, kappa_p2) =
         sif_parameters
     kd = max(kd_p1 * (Tc - T_freeze) + kd_p2, min_kd)
-    x = 1 - J / Jmax
+    x = 1 - J / max(Jmax, eps(FT))
     kn = (kn_p1 * x - kn_p2) * x
-    ϕp0 = kp / (kf + kp + kn)
-    ϕp = J / Jmax * ϕp0
-    ϕf = kf / (kf + kd + kn) * (1 - ϕp)
+    ϕp0 = kp / max(kf + kp + kn, eps(FT))
+    ϕp = J / max(Jmax, eps(FT)) * ϕp0
+    ϕf = kf / max(kf + kp + kn, eps(FT)) * (1 - ϕp)
```

Comment:
I belive it is kd instead of kp. Eq (7) in:

 Raczka, B., Porcar-Castell, A., Magney, T., Lee, J.E., Köhler, P., Frankenberg, C., Grossmann, K., Logan, B.A., Stutz, J., Blanken, P.D. and Burns, S.P., 2019. Sustained Nonphotochemical Quenching Shapes the Seasonal Pattern of Solar-Induced Fluorescence at a High-Elevation Evergreen Forest. Journal of Geophysical Research: Biogeosciences, 124(7), pp.2005- 2020. https://doi.org/10.1029/2018JG004883

Reply:
thanks Renato. let me make a quick PR to fix this